### ISSUE TITLE
Is it possible to program with FT232 programming? Is there a schematic drawing available?

### ISSUE THREAD

**Issue by uguraltinsoy** [MCUdude/MegaCore#203]
Is it possible to program with FT232 programming? Is there a schematic drawing available?

**Comment by MCUdude** [comment]
Yes, you can use any USB to serial chip to upload programs as long as a bootloader is present. This needs to be flashed with an ISP programmer, such as the USBtinyISP or USBasp.

> Is there a schematic drawing available?

Yes, here: https://github.com/MCUdude/MegaCore/tree/master#minimal-setup

**Comment by uguraltinsoy** [comment]
can i use it without bootloader?

http://electronics-diy.com/FT232RL_USB_to_Serial_Adapter_for_PIC_AVR_ATMEGA_ARDUINO_MCUs.php

**Comment by MCUdude** [comment]
> can i use it without bootloader?

Not with MegaCore. You need an ISP programmer to load the bootloader. Then you can use the F232RL to upload new programs.

**Comment by uguraltinsoy** [comment]
Is there a resource on how to load the bootloader, can you guide me?

**Comment by MCUdude** [comment]
There are plenty of resources online. YOu can start by reading the [MegaCore readme file](https://github.com/MCUdude/MegaCore/blob/master/README.md). All you need to do is to connect an ISP programmer to the appropriate pins, select your preferred settings in the IDE Tools menu, and hit "Burn bootloader".
